src/mapml.js

```diff
@@ -944,12 +1048,51 @@ M.MapMLLayer = L.Layer.extend({
                     _leafletLayer: this,
                     crs: this.crs
                   }).addTo(map);
+                  this._setLayerElBounds();
                 }
               }, this);
         }
+        this._setLayerElBounds();
```

Comment:
I'm wondering if line 1055 should be inserted in the if block of line 1033, since it is being unconditionally executed here and may result in errors, consequently?

Reply:
The method is called at the end to assign the layer element itself the .bounds property. The method also needs to run in the cases where `this._extent` doesn't exist like maps that only contain static content, placing it inside of the `if(this._extent)` would prevent that.

As for errors, the method checks certain conditions but there may have been cases that I missed, if any come to mind let me know and I can account for them.